Lab1/Lab1.m

```diff
@@ -0,0 +1,150 @@
+% subtracting one by this result. However, the area underneath the
+% distribution from negative infinity to zero is zero so we are really
+% calculating the area from 0 to z where z is 5.4. Now, we want the
+% probability of producing a signal of 5.4 or larger so we subtract area
+% calculated by the integral of the exponential distribution from negative
+% infinity to 5.4 from one. 
+
+%% Section 3D
+% Calculates the probabilty of background producing a signal.
+expD = makedist("Exponential",'mu',3);
+pTail = 1-cdf(expD,5.4); 
+disp("    "+pTail*100+"%")
+
+%% Section 3E
+% Converts probabilty into equivalent sigma.
+pHead = cdf(expD,5.4);
+disp(norminv(pHead))
+
+%% Section 4
+% Some patterns I noticed from using various hypothetical measurement
+% values is how the probabilities and the sigmas change. Specifically,
+% as the hypothetical measurement increases, the probability decreases
+% while sigma increases. Also, as the hypothetical measurement decreases,
+% the probability increases while sigma decreases. 
+expD = makedist("Exponential",'mu',3);
+pTail = 1-cdf(expD,2.6); 
+disp("    "+pTail*100+"%")
+pHead = cdf(expD,2.6);
+disp(norminv(pHead))
+
+expD = makedist("Exponential",'mu',3);
+pTail = 1-cdf(expD,7.8); 
+disp("    "+pTail*100+"%")
+pHead = cdf(expD,7.8);
+disp(norminv(pHead))
+
+%% Non-Continuous Distributions
+
+%% Section 1B
+% Plots a series of plots with different parameters. One feature I notice
+% is a stair-like plot to prevent noninteger events. For example, it
+% doesn't make sense for 5.4 cellphones to be sold but it makes sense for
+% there to only be 5 or 6 cellphones to be sold. Another feature I notice
+% is the number of instances in x = 0 decrease as lambda increases. To add
+% on to this, there are the most instances lambda. 
+x = 0:10;
+poisson = makedist("Poisson",'lambda',1); 
+subplot(1,2,2)
+stairs(x-0.5,pdf(poisson,x),"LineWidth",2);
+set(gca,'YScale','log')
+subplot(1,2,1)
+stairs(x-0.5,pdf(poisson,x),"LineWidth",2); figure;
+
+x = 0:10;
+poisson = makedist("Poisson",'lambda',6); 
+subplot(1,2,2)
+stairs(x-0.5,pdf(poisson,x),"LineWidth",2);
+set(gca,'YScale','log')
+subplot(1,2,1)
+stairs(x-0.5,pdf(poisson,x),"LineWidth",2); 
+
+%% Section 1C
+% My hypothetical question is "What is the probability of the background
+% producing a signal of 3.2?". Calculates the corresponding probability. 
+poissonD = makedist("Poisson",'lambda',3.2);
+pTail = 1-cdf(poissonD,3.2); 
+disp("    "+pTail*100+"%")
+pHead = cdf(poissonD,3.2);
+disp(norminv(pHead))
+
+%% Section 1D
+% Since the distributions are discrete, the probabilities and sigmas are
+% also discrete. The implication of this in an experiment is how we only 
+% expect certain values. For example, we will expect integer numbers and
+% not numbers like 1.2, 5.28, and the like. Also, we know that the
+% distribution will list every possible outcome and that the sum of all the
+% probabilities will be equal to one.  
```

Comment:
as long as the distribution is normalized

Reply:
Should I specifically say that it will be equal to one if the distribution is normalized? I think all of them are normalized in this Lab but I'm not sure... I can also just omit this but I'm not sure what to say otherwise.

Reply:
you are right they are all normalized so you should be good